src/components/AnonymousChat/child/ChannelImage.tsx

```diff
@@ -238,7 +238,7 @@ const ChannelImage = ({
   if (type === BaseChannelItemTypeProps.MY_ANON_POST_NOTIFICATION) {
     return (
       <View>
-        <FastImage source={AnonymousProfile} style={styles.image} />
+        {renderMainImage()}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to my previous comment, the change from using `FastImage` directly to a function call `renderMainImage()` seems like a good abstraction. But again, without seeing the implementation of `renderMainImage()`, I can't fully evaluate this change. If `renderMainImage()` handles all the previous responsibilities of the `FastImage` component (like handling loading states, errors, etc.), then this change should be fine.


<!-- This is an auto-generated comment by OSS CodeRabbit -->